libs/s3/objects.go

```diff
@@ -52,3 +58,25 @@ func (s Client) ListObjects(prefix, delimiter string) (*s3.ListObjectsOutput, er
 
 	return result, nil
 }
+
+// Load loads the risk levels json from S3 and returns a slice of bytes
+func (s Client) Load(l Loader, path string) error {
+	if len(path) == 0 {
+		return errors.New("Path cannot be empty")
+	}
+
+	log.Printf("Loading %s", path)
+	o, err := s.GetObject(path)
+	if err != nil {
+		if len(o) == 0 {
+			return errors.New("Object not found in S3")
+		}
+		return errors.New("Unable to get object from S3")
+	}
+
+	if err := l.Load(o); err != nil {
+		return err
+	}
+
+	return nil
+}
```

Comment:
lol now you have 3 `Load()` methods!  Make this take `interface{}` instead.

Reply:
I assumed that wasn't gonna work without even trying it 😁 Much better now!